exo-19-json.js

```diff
@@ -0,0 +1,99 @@
+var JOURNAL = [
+    {"events":["carrot","exercise","weekend"],"squirrel":false},
```

Comment:
Use https://www.w3schools.com/js/js_json_stringify.asp to do it


Utilisez https://www.w3schools.com/js/js_json_stringify.asp pour le faire